src/components/__tests__/SearchResults.test.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
```

Comment:
we should look into snapshots for stuff like this

Reply:
I think snapshots will come in handy once we have a pretty good handle on the final output of components, but until then we'll just get snapshot warnings over and over while we work out the final UI.